packages/core/services/FileDownloadService/index.ts

```diff
@@ -1,4 +1,6 @@
 import axios from "axios";
+import * as http from "http";
+import * as https from "https";
```

Comment:
`http` & `https` are node libraries so we can't use them in the web version of the app, any logic that requires these libraries has to go into the `desktop` package. Probably easier to find a web friendly way to do this instead.

https://nodejs.org/api/http.html

Reply:
resolved in https://github.com/AllenInstitute/biofile-finder/pull/252!